controller/deployments.go

```diff
@@ -259,11 +259,15 @@ func (g *defaultClientGetter) GetKubeClient(ctx context.Context) (kubernetes.Kub
 		return nil, errs.Wrap(err, "could not retrieve namespace name")
 	}
 
+	// TODO replace with configurable timeout
```

Comment:
Please add a comment to clarify that this is a per-HTTP request from the backend to k8s, and that the Hawkular (and any other you know) API ignores this.

Reply:
Good suggeston! I've added a comment. Let me know if this needs further clarification.